etc/cloud_nightly.yml

```diff
@@ -1 +1,48 @@
-# Evergreen configuration for MongoDB Cloud E2E tests with MongoDB
+stepback: true
+
+# Needed til EVG-3714 is deployed
+functions:
+  something:
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |
+        exit
+
+tasks:
+- name: GENERATE_MMS_TESTS
+  commands:
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |
+        set -ex
+        git clone --branch ${cloud_branch} --depth 1 git@github.com:${cloud_repo} mms
+        cd mms/scripts
+        ./init_virtualenv.sh ${workdir} 'no-om'
+        VIRTUAL_ENV_DISABLE_PROMPT=true source venv-activate
+        cd evergreen
+        sleep ${compile_wait_duration_seconds}
```

Comment:
I was under the impression that you were going to poll to wait for the compile task to run if it hadn't already. Is that going to happen as a follow-up pull request?

It seems like it'd be beneficial to have a separate script in the `mongo_cloud_nightly/` directory that is responsible for waiting for the compile task to have run so the implementation can be changed without needing to make further changes to this embedded bash script.

Reply:
Yeah the polling happens externally because I set it up so that tasks can be generated for multiple server buildvariants if desired, and each one would have a different compile task that would need to be waited on. If you have access to the cloud repo I can show you.

The interval in this repo is more about a common sense minimum duration to apply to all cases, and to avoid any race condition where it will try to generate tasks that wait on tasks that have not been created yet because Evergreen dispatched tasks for this project before the server project itself got a new version created in Evergreen, and to reduce the time there would need to be potentially multiple Evergreen hosts waiting on tasks, if we ever have a case where we are using multiple buildvariants of the server.

Reply:
Got it, thanks for the detailed explanation.